bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -2867,7 +2870,9 @@ ReplicaImp::ReplicaImp(bool firstTime,
       metric_sent_fullCommitProof_msg_due_to_reqMissingData_{
           metrics_.RegisterCounter("sentFullCommitProofMsgDueToReqMissingData")},
       metric_not_enough_client_requests_event_{metrics_.RegisterCounter("notEnoughClientRequestsEvent")},
-      metric_total_finished_consensuses_{metrics_.RegisterCounter("totalOrderedRequests")} {
+      metric_total_finished_consensuses_{metrics_.RegisterCounter("totalOrderedRequests")},
+      metric_total_slowPath_{metrics_.RegisterCounter("totalSlowPaths")},
```

Comment:
How does `totalSlowPaths` differ from the existing `slowPathCount` ? Aren't those redundant?

Reply:
The slowPathCounts seems to be not accurate. (we have seen in the dashboards that sometimes sloePathCount > lastExecuteSeqNum which makes no sense.
I've prefered not to touch slowPathCount as there are tests that rely upon its value. and instead, I've added the totalSlowPaths which count the slow path only after the execution phase.

Reply:
Thanks, we'll also have to switch the fast/slow path Apollo tests to your new metrics. They seem to make more sense.